test/infra.go

```diff
@@ -0,0 +1,42 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package test
```

Comment:
Both this file and the one above belong to tool. It might make more sense to move the entire test package into the tool directory and treat them as tests of tool.

Reply:
ok, then, maybe it would be good to add some comments or description. I was a bit confused at the beginning. I had to read the code to understand what they are doing and that they invoke the `tool`.